packs/trendmicro-cloudone-conformity/config.yml

```diff
@@ -0,0 +1,34 @@
+
+# Name of the pack (required)
+name: trendmicro-cloudone-conformity
+
+title: New Relic’s integration with Trend Micro Cloud One - Conformity
+
+# Description of the pack (required)
+description: >-
+  New Relic’s integration with Trend Micro Cloud One - Conformity ingests cloud security posture management (CSPM)
+  data from Trend Micro Cloud One - Conformity into New Relic in real-time. 
+  The integration deploys a Amazon Web Services (AWS) CloudFormation stack in your AWS account. 
+  Using this integration, our AWS, Microsoft Azure and Google Cloud Platform (GCP) customers can bring in their 
+  Conformity generated CSPM data into New Relic One where it's contextualized and correlated with workload telemetry data 
+  delivering AI powered visualizations and quick insights.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
```

Comment:
I'm fairly certain all Partner QSs should be labeled `Verified` can you confirm @polfliet ?

Based on our definition found in `G-drive:` `Observability Packs: Support Model Requirements`

Reply:
This is an [experimental repo](https://github.com/newrelic-experimental/newrelic-connector-trendmicro-cloud-one-conformity), and hence I think it should be "Community".

Reply:
Good point on why we'd leave it labeled as a `Community` QS. Thanks

Reply:
Sounds good thanks for clarifying.